### ISSUE TITLE
Incorrect ORCID for Geoffrey Hall

### ISSUE THREAD

**Issue by peterdesmet** [dshorthouse/collector#1]
Hi @dshorthouse, just browsed a bit around in collector and stumbled upon http://collector.shorthouse.net/agent/0000-0002-6299-8385. I think that Geoffrey Hall (Canada, botanist) is not the one with https://orcid.org/0000-0002-6299-8385 (UK, physics). In other words, those are two different Geoffs. 😄

**Comment by dshorthouse** [comment]
Not surprised. This is a prime example of the dangers of blindly using an API. At the time I built this proof-of-concept several years ago, there was only one Geoffrey Hall in ORCID. To do this properly at the scale of GBIF, you MUST make use of the ORCID OAuth2 authentication and logged-in users then verify their identities and thus definitively claim their previously private profiles. One should not, as I did, use the ORCID API to search for people, cache those ORCIDs, & make public. FWIW, I removed the ORCID for Geoff.
